src/osd/OSD.h

```diff
@@ -595,13 +597,13 @@ class OSDService : public Scrub::ScrubSchedListener {
   void queue_scrub_event_msg(PG* pg,
 			     Scrub::scrub_prio_t with_priority,
 			     unsigned int qu_priority,
-			     Scrub::act_token_t act_token);
+			     Scrub::act_token_t act_token,
+                             int64_t cost_per_object);
 
   /// An alternative version of queue_scrub_event_msg(), in which the queuing priority is
   /// provided by the executing scrub (i.e. taken from PgScrubber::m_flags)
   template <class MSG_TYPE>
-  void queue_scrub_event_msg(PG* pg, Scrub::scrub_prio_t with_priority);
-  int64_t get_scrub_cost();
+  void queue_scrub_event_msg(PG* pg, Scrub::scrub_prio_t with_priority, int64_t cost_per_object);
```

Comment:
Most of the changes above are due to there only being two users that actually need to pass cost_per_object.  For the rest, add a queue_simple_scrub_event_msg variant that calls queue_scrub_event_msg with osd_scrub_event_cost.

Reply:
Yeah makes sense, I'll make the change